cmd/tsgo/sys.go

```diff
@@ -46,11 +46,6 @@ func (s *osSys) Writer() io.Writer {
 	return s.writer
 }
 
-func (s *osSys) EndWrite() {
```

Comment:
Very happy to see `EndWrite` go

Reply:
Yeah that one revealed bug when writing diagnostics that dont have locations